draft-ietf-quic-transport.md

```diff
@@ -712,12 +743,13 @@ packet types explicitly indicate the encryption level and therefore the keys
 that are used to remove packet protection.
 
 Packets protected with 0-RTT keys use a type value of 0x7C.  The connection ID
-field for a 0-RTT packet is selected by the client.
+fields for a 0-RTT packet MUST match the values used in the Initial packet
+({{packet-initial}}).
 
 The client can send 0-RTT packets after receiving a Handshake packet
 ({{packet-handshake}}), if that packet does not complete the handshake.  Even if
 the client receives a different connection ID in the Handshake packet, it MUST
-continue to use the connection ID selected by the client for 0-RTT packets, see
+continue to use the same Destination Connection ID for 0-RTT packets, see
```

Comment:
What should a receiver do if a subsequent packet is received with a different source connection ID?

Reply:
I added text that says that you discard those.